src/main/scala/cromwell/engine/file.txt

```diff
@@ -0,0 +1,1000 @@
+| operations/ENniofmMKhiE8ZnogOjH1kEgw7vetLsXKgpwcm9kdWN0aW9u   |
```

Comment:
gah that was not supposed to be checked in.  These are some IDs from my 1KWH testing today that needed some investigating in JES-land.